### DISCUSSION TITLE
Windows 生成compile_commands.json，盘符小写vscode clangd不能跳转源文件，要改为大写

### DISCUSSION THREAD

**Discussion by tghul** [xmake-io/xmake#4602] [General]
Windows 生成compile_commands.json，盘符小写vscode clangd不能跳转源文件，要改为大写
[https://discuss.em-ide.com/d/503-clangd](https://discuss.em-ide.com/d/503-clangd)
[clangd/clangd#108](https://github.com/clangd/clangd/issues/108)
[clangd/clangd#102 (comment)](https://github.com/clangd/clangd/issues/102#issuecomment-515081564)

**Comment by waruqi** [comment]
所以这不应该是 clangd 的问题么，坐等他们修复
